.gitignore

```diff
@@ -136,3 +136,6 @@ dmypy.json
 
 # direnv
 .envrc
+
+# Ignore version module generated by setuptools_scm
+/*/_version.py
```

Comment:
I think this `*`  should be ok here (one less thing for downstream projects to update)